enchant.lua

```diff
@@ -694,13 +688,11 @@ function HandleEnchantCommand( Split, Player )
 		return true
 	end
 
-	local Result, ReturnMessage, Enchantment = EnchantItem( Split )
+	local ReturnMessage, ItemName, EnchantmentName, PlayerName = EnchantItem( Split )
```

Comment:
I think it would be best if the `EnchantItem` function had explicit parameters for the player name, enchantment name and level, rather than a generic split. You're already refactoring the function, so it's best to apply the fix now, too.